pricegraph/src/orderbook.rs

```diff
@@ -578,11 +586,9 @@ impl Orderbook {
         Some((order, user))
     }
 
-    /// Returns `true` if the specified token pair is valid, and `false`
-    /// otherwise.
     fn is_token_pair_valid(&self, pair: TokenPair) -> bool {
-        let max_token = (self.projection.node_count() - 1) as u16;
-        pair.buy <= max_token && pair.sell <= max_token
+        let node_count = self.projection.node_count();
+        (pair.buy as usize) < node_count && (pair.sell as usize) < node_count
```

Comment:
This change isn't needed but at first I thought the `- 1` was a mistake until I realized that there is always a fee token so node_count is never 0. I find this way clearer.
Removed the doc comment because it doesn't add information to the signature.

I don't mind reverting this if we disagree, just included it because I came across it.

Reply:
I'm fine with this change, I also agree it is a little more readable this way.